language/move-lang/src/expansion/ast.rs

```diff
@@ -46,12 +46,24 @@ pub struct Script {
 pub struct ModuleDefinition {
     pub loc: Loc,
     pub is_source_module: bool,
+    pub friends: UniqueMap<ModuleIdent, Friend>,
     pub structs: UniqueMap<StructName, StructDefinition>,
     pub functions: UniqueMap<FunctionName, Function>,
     pub constants: UniqueMap<ConstantName, Constant>,
     pub specs: Vec<SpecBlock>,
 }
 
+//**************************************************************************************************
+// Friends
+//**************************************************************************************************
+
+#[derive(PartialEq, Clone, Debug)]
+pub enum Friend_ {
+    Ok,
+    UnresolvedError,
+}
+pub type Friend = Spanned<Friend_>;
+
```

Comment:
Remove this. If you hit an error, just drop it from the map

Reply:
Cleaned up this and the one in naming.